cfgov/prepaid_agreements/jinja2/prepaid_agreements/index.html

```diff
@@ -0,0 +1,100 @@
+{% extends 'layout-full.html' %}
+
+{# HEAD items #}
+
+{% block title -%}
+    Prepaid Agreements Search | Consumer Financial Protection Bureau
+{%- endblock title %}
+
+{% block css -%}
+    {{ super() }}
+    <link rel="stylesheet" href="{{ static('apps/prepaid-agreements/css/main.css') }}">
+{%- endblock css %}
+
+{# BODY items #}
+
+{% block content_main_modifiers -%}
+    content__1-3 prepaid-agreements prepaid-agreements-search
+{%- endblock %}
+
+{% block content_main scoped %}
+    {%- import 'molecules/breadcrumbs.html' as breadcrumbs with context -%}
+    {% set breadcrumb_items = [{
+        'href': '/data-research/prepaid-agreements/',
+        'title': 'Prepaid account agreements'
+    }]%}
+    {{ breadcrumbs.render(breadcrumb_items) }}
+
+    <div class="content_wrapper search_form">
+        <div class="block block__sub search_intro">
+            <h1>Prepaid account agreements database</h1>
+            <div class="lead-paragraph u-readable-width">
+                Find prepaid product agreements from more than 80 issuers. You can search agreements by issuer name, product name, prepaid account type, and other critera.
```

Comment:
Are there any plans to make this "80" dynamic? Will this get out of date over time?

Reply:
We don't expect this number to change significantly, so at this point it's probably not necessary to make it dynamic. We can revisit later if this changes, though.